README.md

```diff
@@ -4,6 +4,10 @@ Hello and welcome! Hopefully this can help anyone get up and running with Frozen
 
 Issues are welcome but pull requests are even more welcome if you can!
 
+## Specials thanks
```

Comment:
Thanks for adding this!

Reply:
This part is fine ? Or do you want I add something ?